addons/spark/0.x/spark.yaml

```
@@ -0,0 +1,31 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: Addon
+metadata:
+  name: spark
+  labels:
+    kubeaddons.mesosphere.io/name: spark
+    kubeaddons.mesosphere.io/hack-requires-defaultstorageclass: "true"
+  annotations:
+    stage.kubeaddons.mesosphere.io/spark: Beta
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.1.0-2"
+    appversion.kubeaddons.mesosphere.io/spark: "0.2.0"
+spec:
+  kubernetes:
+    minSupportedVersion: v1.16.0
```

This is optional, but you are OK to be more restrictive with this version if you like, for instance:
```suggestion
    minSupportedVersion: v1.17.0
```
I don't know if that makes sense for this app, but I just wanted to point it out in case you had only tested it in 1.17, e.t.c.

---

we are actually testing it on konvoy 1.1.5 , so I think we can set it to 1.15